src/views/states/list/hooks/useStateFilters.ts

```diff
@@ -9,6 +9,28 @@ const useStateFilters = (): RowFilter<V1beta1NodeNetworkState>[] => {
   const { t } = useNMStateTranslation();
 
   return [
+    {
+      type: FILTERS_TYPES.IP_ADDRESS,
+      filter: (searchText, obj) => {
+        const searchIpAddress = searchText?.selected?.[0];
```

Comment:
By our conventions, you should capitalize all letters in an abbreviation unless it's the abbreviation comes first in the symbol. So, this should be `searchIPAddress`.